core/templates/pages/signup-page/signup-page.component.html

```diff
@@ -166,4 +166,10 @@
   .oppia-checkbox-margin {
     margin-right: 6px;
   }
+
+  .radio label {
+    display: flex;
```

Comment:
Do we need this? If yes, what it fixes?

Reply:
It adds space between the radio button and label. I made it to flex and then added gap.

Reply:
Sorry, I wasn’t specific enough. This exact line with display: flex, do we need it? Will the solution work without it?

Reply:
The problem is that the gap between radio and label is very less which does not look aesthetic so I have increased the space between them.

Reply:
I understand the problem, and I understand the 172 line. Will the solution work if you remove the line 171? What value does the 171 line bring?

Reply:
@nikitaevg the solution does not work if the display is not set to flex